proto/build.gradle.kts

```diff
@@ -52,11 +52,11 @@ dependencies {
 protobuf {
     // Configure the protoc executable
     protoc {
-        artifact = "com.google.protobuf:protoc:3.5.0"
+        artifact = "com.google.protobuf:protoc:${libs.versions.protobuf.get()}"
```

Comment:
Force protoc version to mirror whatever protobuf version is chosen for parity's sake